src/Microsoft.Extensions.FileProviders.Physical/PhysicalFileProvider.cs

```diff
@@ -291,5 +272,29 @@ public IChangeToken Watch(string filter)
 
             return _filesWatcher.CreateFileChangeToken(filter);
         }
+
+        private static IDirectoryContents GetEnumerableContents(string fullPath)
+        {
+            return new EnumerableDirectoryContents(() =>
```

Comment:
Could we fork it and make a `PhysicalDirectoryContents`? There's not much to implement in the interface.